src/sentry/integrations/github/webhook.py

```diff
@@ -86,18 +91,74 @@ def is_contributor_eligible_for_seat_assignment(user_type: str | None) -> bool:
     return user_type != "Bot"
 
 
+def _handle_pr_webhook_for_autofix_processor(
```

Comment:
This function is a wrapper for `handle_github_pr_webhook_for_autofix` in order to have the same signature for all webhook processors.

This allows using the top-level `_handle`/ `WEBHOOK_EVENT_PROCESSORS`.

<img width="1066" height="242" alt="Image" src="https://github.com/user-attachments/assets/93f74990-2ad3-4aac-9fc4-95cdf36178ad" />

<img width="744" height="122" alt="Image" src="https://github.com/user-attachments/assets/95aa8c4c-5928-462c-a28f-5e8948231e3e" />